catalogs/catalog-hive/src/test/java/com/datastrato/gravitino/catalog/hive/integration/test/CatalogHiveIT.java

```diff
@@ -809,6 +810,36 @@ public void testAddPartition() throws TException, InterruptedException {
     Assertions.assertEquals(2, count);
   }
 
+  @Test
+  public void testDropPartition() throws TException, InterruptedException {
+    Table createdTable = preparePartitionedTable();
+
+    // add partition "hive_col_name2=2023-01-02/hive_col_name3=gravitino_it_test2"
+    String[] field1 = new String[] {"hive_col_name2"};
+    String[] field2 = new String[] {"hive_col_name3"};
+    Literal<?> literal1 = Literals.dateLiteral(LocalDate.parse("2023-01-02"));
+    Literal<?> literal2 = Literals.stringLiteral("gravitino_it_test2");
+    Partition identity =
+        Partitions.identity(new String[][] {field1, field2}, new Literal<?>[] {literal1, literal2});
+    IdentityPartition partitionAdded =
+        (IdentityPartition) createdTable.supportPartitions().addPartition(identity);
+
+    // test drop partition "hive_col_name2=2023-01-02/hive_col_name3=gravitino_it_test2"
+    boolean dropRes = createdTable.supportPartitions().dropPartition(partitionAdded.name(), true);
+    Assertions.assertTrue(dropRes);
```

Comment:
Please test read/write before drop, and test HDFS file after drop

Reply:
done